packages/design-system/src/components/Table/Table.tsx

```diff
@@ -93,7 +160,7 @@ export class Table extends React.PureComponent {
     });
   }
 
-  render() {
+  render(): React.ReactNode {
```

Comment:
Based on the [original type definition file](https://github.com/CMSgov/design-system/pull/1088/files#diff-3afd608cb36c07146ec6a790493d6b24b8e1341d6661aed7498a1ae5be468f5aL59), should this be `JSX.Element`?

Reply:
I've noticed that our other typescript conversion uses `React.ReactNode` and have used the same. Also this [slackoverflow](https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement#58123882) has more info.